.travis.yml

```diff
@@ -10,4 +10,4 @@ before_script:
 script:
   - pytest --capture=sys
 after_success:
-  - coveralls 
+  - codecov 
```

Comment:
The title of this pull request should be changed to something like ".travis.yml: Removed coveralls and added codecov" to make it easier to understand the objective.  Othewise, looks great.  Nice work.